bitcoin/src/crypto/key.rs

```diff
@@ -237,18 +236,18 @@ impl fmt::Display for PublicKey {
 impl FromStr for PublicKey {
     type Err = ParsePublicKeyError;
     fn from_str(s: &str) -> Result<PublicKey, ParsePublicKeyError> {
-        use HexToArrayError::*;
+        use hex_stable::ToArrayError::*;
```

Comment:
Example usage of the inner error type being discussed [here](https://github.com/rust-bitcoin/hex-conservative/pull/162#issuecomment-2730511712). 

This post is just so I can link to the code.